sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/GcpApiSurfaceTest.java

```diff
@@ -75,6 +75,8 @@ public void testGcpApiSurface() throws Exception {
             classesInPackage("javax"),
             classesInPackage("org.apache.beam"),
             classesInPackage("org.apache.commons.logging"),
+            classesInPackage("com.google.cloud"),
```

Comment:
Unfortunately, the unit tests fail without this line, ostensibly due to the fact that Timestamp, Date, etc.  are located in the com.google.cloud package.